### ISSUE TITLE
Drag and drop sorting with multiple pages of results

### ISSUE THREAD

**Issue by JedWatson** [keystonejs/keystone#1636]
(initially discussed in #1633)

At the moment we don't have a good way of handling list sorting when you've got a large set of items.

One option is to allow customisation of the number of items per page to allow for sorting large lists (or possibly add a "show all" button to the pagination since the performance is pretty good, within reasonable limits)

Another idea raised by @cantuket is to allow dragging over a page, and navigating to that page to allow the moving of items across pages; sorting across pages requires a completely different back-end implementation from our current one though.

I'm putting this on the post-0.4 roadmap for further discussion.

**Comment by gopeter** [comment]
We've built a similar solution for a client with large datasets (list with ~100.000 entries) who wanted an option to assign entry A, B, C, .., n to entry X (like nesting parent/child records). We've built a pagination (limit per page: 1000) and a "clipboard". Click on "copy"-button at entry A copies this one to the clipboard (a toggelable div container), click on "move"-button at entry B assigns all the entries from the clipboard to entry B. Could be much prettier, but it works :) Doesn't matter which page you choose, assigning works across pages.

I think some kind of this clipboard-idea could help to implement a sorting feature across pages.

![clipboard](https://cloud.githubusercontent.com/assets/1242960/10477685/820125d2-7258-11e5-80d9-90c44c66b30b.png)
